dragscroll.js

```diff
@@ -90,3 +95,16 @@
     exports.reset = reset;
 }));
 
+function zoomIn(n) {
+    zoom = zoom + n;
+    sheet.innerHTML = "#dragsimg {width:" + zoom + "%;}";
+    document.body.appendChild(sheet);
+}
+function zoomOut(n) {
+    if(zoom<=100){
+        return false;
+    }
+    zoom = zoom - n;
```

Comment:
Better check here `zoom = Math.max(0, zoom - n);` or something like that, depending on what you mean